reference/toolstacks/nodejs/README.md

```diff
@@ -20,59 +20,70 @@ To use Platform.sh and Node.js together, configure the ``.platform.app.yaml``
 file with a few key settings, as described here (a complete example is included
 at the end).
 
-1. Specify the `type` as `nodejs` you should definitely specify a version for example `type: nodejs:4.2`
+1. Specify the `type` as `nodejs`. You may also specify a version for example `type: nodejs:4.4`.
 2. Specify your dependencies under the `nodejs` key, like this:
 
-```yaml
- dependencies:
-   nodejs:
-     pm2: "^0.15.10"
-```
+   ```yaml
+   dependencies:
```

Comment:
To have continued number list .. 1. 2. 3. 4. .... See the current page... the number list is broken.